src/types/redis_json.cc

```diff
@@ -91,6 +91,14 @@ rocksdb::Status Json::Info(const std::string &user_key, JsonStorageFormat *stora
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status Json::create(const std::string &ns_key, JsonMetadata metadata, const std::string &value) {
```

Comment:
```suggestion
rocksdb::Status Json::create(const std::string &ns_key, JsonMetadata &metadata, const std::string &value) {
```

I prefer lvalue reference or pointer since it will be changed in `write`.

Reply:
yes, it's a good point.